.github/workflows/build.yml

```diff
@@ -16,9 +16,9 @@ jobs:
   Build:
     uses: microsoft/digitalworkplace-workflows/.github/workflows/build.yml@v3.1
     with:
-      project-to-build: "src/Generator/Generator.csproj"
-      project-to-test: "test/Generator.Tests/Generator.Tests.csproj"
-      coverage-threshold: 89
```

Comment:
Any reason for un-denting by a space?

Reply:
Nope. Looks like an accidental change. I've updated it.